src/templates/multipleClaimWarning.md

```diff
@@ -0,0 +1,3 @@
+Hello @{commenter}, it looks like someone is already assigned to this issue. We believe multiple assignments to the same issue causes confusion, and hence encourage you to go through other issues in this repository or other Zulip repositories and start working on them. Meanwhile, you can always return here and claim this issue in case it gets unassigned!
```

Comment:
@rishig Does this comment template look alright to you? Feel free to suggest changes if more improvement are needed